be2-scala/src/main/scala/ch/epfl/pop/storage/DbActor.scala

```diff
@@ -199,6 +200,31 @@ final case class DbActor(
     }
   }
 
+  //generates the key of the RollCallData to store in the database
+  private def generateRollCallDataKey(laoId: Hash): String = {
+    s"${ROLL_CALL_DATA_PREFIX}${laoId.toString}"
+  }
+
+  @throws [DbActorNAckException]
+  private def readRollCallData(laoId: Hash): RollCallData = {
+    Try(storage.read(s"${ROLL_CALL_DATA_PREFIX}${laoId.toString}")) match {
```

Comment:
can you use generateRollCallDataKey since you defiend it in this case? So that if we change it for some reason, no need to modify multiple instances :)